src/lambdas/initiateDataRequest/zendeskTicketDiffersFromRequest.ts

```diff
@@ -126,12 +154,9 @@ const ticketAndRequestDetailsDiffer = (
     unmatchedParameters.push('recipientName')
   if (!matchStringParams(requesterDetails.email, requestParams.requesterEmail))
     unmatchedParameters.push('requesterEmail')
+  // TODO: add check for date matching
```

Comment:
Does this need doing in this PR? Notice upi jabe a "datesMatch" function you're using above!

Reply:
Yep, I had already done it, so I've removed the TODO